poetry/templates/singular.html

```diff
@@ -0,0 +1,31 @@
+<!-- wp:template-part {"slug":"header","theme":"community-themes/poetry"} /-->
```

Comment:
```suggestion
<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
```
We need the header tagName for accessibility, and we need to remove the theme slug because that route may not be the same for every installation